internal/printer/printer.go

```diff
@@ -20,16 +20,69 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strings"
 
 	"github.com/GoogleContainerTools/kpt/internal/types"
 )
 
+const (
+	// FnIndentation is the number of spaces at the beginning of each line of
+	// function running progress.
+	FnIndentation = 0
+)
+
+// DisableOutputTruncate defines should output be truncated
+var DisableOutputTruncate bool
```

Comment:
Does this have to be a package-level variable? Why don't we just pass this in when we create the printer?

Reply:
We create the printer during creating command so we cannot use the flag value at that time?